app-interface/manifests/rhose-flee-manager.yml

```diff
@@ -0,0 +1,126 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: rhose-fleet-manager
+  labels:
+    app: rhose-fleet-manager
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: rhose-fleet-manager
+      labels:
+        app-component: rhose-fleet-manager
+    spec:
+      selector:
+        matchLabels:
+          app-component: rhose-fleet-manager
+      replicas: ${{FLEET_MANAGER_DEPLOYMENT_REPLICAS}}
+      template:
+        metadata:
+          labels:
+            app-component: rhose-fleet-manager
+        spec:
+          containers:
+            - image: ${{FLEET_MANAGER_IMAGE}}:${{{FLEET_MANAGER_IMAGE_TAG}}}
+              imagePullPolicy: IfNotPresent
+              name: rhose-fleet-manager
+              resources:
+                limits:
+                  cpu: ${{FLEET_MANAGER_CPU_LIMITS}}
+                  memory: ${{FLEET_MANAGER_MEM_LIMITS}}
+                requests:
+                  cpu: ${{FLEET_MANAGER_CPU_REQS}}
+                  memory: ${{FLEET_MANAGER_MEM_REQS}}
+              ports:
+                - containerPort: 8080
+              envFrom:
+                - configMapRef:
+                    name: fleet-manager-bridge-config
```

Comment:
@msmagnanijr Needs to be `rhose-fleet-manager-config` and `rhose-fleet-manager-secrets`